examples/data-objects/client-ui-lib/package.json

```diff
@@ -78,7 +78,7 @@
     "@types/assert": "^1.5.2",
     "@types/debug": "^4.1.5",
     "@types/jsdom": "^12.0.0",
-    "@types/mocha": "^5.2.5",
+    "@types/mocha": "^8.2.2",
```

Comment:
any chance you can do these bumps as separate PR?

Reply:
> Given that this is not Fluid-specific, why should it be part of Fluid repo? Should it be stand-alone package?

Craig is kindly offering to contribute his harness to Fluid to help us begin automated performance testing, which is something Curtis and I had expressed a desire for.

The fact that Craig packaged it so nicely as a stand-alone/reusable package with standardized reporting is an unexpected bonus and does open the possibility of maintaining the package separately.

I'm torn.  Part of me selfishly wants it in 'Fluid-Experimental' so it's convenient to tweak while bringing up our first few benchmarks, but it's probably a better deal for @CraigMacomber to host it independently so his commits aren't gated on Fluid developers and processes.

@CraigMacomber - Let me talk this over with @curtisman during our Tuesday sync and see if he has a desire for this to be an "in house" tool.  If we do publish it independently, I can help whip up a public GH repo with automated CI.